sp/src/game/server/hl2/item_healthkit.cpp

```diff
@@ -113,6 +119,7 @@ void CHealthKit::Precache( void )
 		m_nSkin = 1;
 
 	PrecacheScriptSound( pTouchSounds[ GetEZVariant() ] );
+	PrecacheScriptSound( "HealthVial.BackpackPickup" );
```

Comment:
This precaches whether `sv_healthvial_backpack` is enabled or not. I'm assuming that's intentional

Reply:
That's a good point, but if it did check sv_healthvial_backpack then the sound wouldn't be cached if the value gets changed at any point after the map has loaded